### ISSUE TITLE
Hbs blog template does not show images

### ISSUE THREAD

**Issue by jstockwin** [keystonejs/generator-keystone#173]
The [jade blog template](https://github.com/keystonejs/generator-keystone/blob/master/app/templates/templates/default-jade/views/blog.jade#L20) includes the image for the blog post (if there is one), however this does not happen in the [handlebars template](https://github.com/keystonejs/generator-keystone/blob/master/app/templates/templates/default-hbs/views/blog.hbs). 

I feel like they should both do the same thing, and showing the image seems nice.

This is also the case in the post templates themselves - jade automatically adds the image to the bottom of the post whereas the handlebars template does not.  

Happy to PR a fix, although would appreciate some pointers on the hbs equivalent of jade's `img.fit(160, 160)` if one exists.

**Comment by pll33** [comment]
There's a [CloudinaryUrl helper here](https://github.com/keystonejs/generator-keystone/blob/master/app/templates/templates/default-hbs/views/helpers/index.js#L154). 

The hbs equivalent would be something like: 
`{{{cloudinaryUrl image width=160 height=160 crop='fit' }}}`

**Comment by jstockwin** [comment]
Perfect, thanks - I'll PR a fix for this later.

**Comment by mxstbr** [comment]
Awesome, thanks @jstockwin!

**Comment by jstockwin** [comment]
@mxstbr Silly question: I think I've made the required changes for this, but how do I actually test this locally? I tried running `node test/test-creation.js` but got `ReferenceError: describe is not defined` at `test/test-creation.js:8:1`.

I'm pretty sure my changes are correct, but obviously they need to be tested. If I run `yo keystone`, it's just going to use the npm version...

**Comment by mxstbr** [comment]
*Assuming you have the `keystone` repositories cloned locally and a generated instance of a project in a folder called `myProject`*

1) In the `keystone` repo folder, run `$ npm link`. This will link that Keystone as if it was globally installed, and allows you to

2) Run `$ npm link keystone` in the `projectName` folder, which will link make npm use the global installation of keystone as the local one.

Since the global installation of Keystone now points to your local folder, whenever you change something there you'll see the changes reflected in the browser! (Assuming you ran `npm start` in the `projectName` folder)

Let me know if this was unclear!

**Comment by jstockwin** [comment]
Okay, thanks. Although it's generator-keystone that I'm testing here, not just base keystone, so presumably I also need to run `npm link` in the generator-keystone repo folder?

**Comment by mxstbr** [comment]
No, since what version of keystone the generated project uses matters, not the generator. ✌️

**Comment by jstockwin** [comment]
Even though the files I am changing are the generator template files?

**Comment by mxstbr** [comment]
Yeah, since the generator only "copies" those files to a new folder to generate the project.

**Comment by jstockwin** [comment]
Sorry, but I'm still not getting this to work. 

I have a local copy of both the `keystone` and the `keystone-generator` repos, the latter of which I have updated and would like to test.

I ran `npm link` in the keystone repo, and then moved to a folder `devSite`. Running `yo keystone` in that still gives me template files from the version of `keystone-generator` on npm, rather than my local version? 

Anyway, I've just copied the new template files from the generator repo and it's working, so will make a PR now. How do I go back to using npm keystone...?
